android/brave_java_resources.gni

```diff
@@ -235,6 +235,7 @@ brave_java_resources = [
   "java/res/drawable-nodpi/ont.png",
   "java/res/drawable-nodpi/pax.png",
   "java/res/drawable-nodpi/rvn.png",
+  "java/res/drawable-nodpi/sol.png",
```

Comment:
I think I commented that we need to remove that png. No reason to add extra resources, we have a vector image of Solana, why do we need the png?

Reply:
I used `png` there because ETH was PNG and desktop is all pngs. If needed they can be changed to `svg` but don't we have a compatibility problem on older OS?

Reply:
It's fine, let's leave sol.png inside makeNetworkAsset, it's for the future if a component updater will serve it, but we don't need to have any sol.png locally. Where do you use it?

Reply:
my prior PRs where it shows accounts, assets, edit visible assets are not using the png at all, they all use a vector image.

Reply:
@wchen342 what you did is right, we serve eth.png via component updater, we don't use it from our local resources. Our local resources are for a diff purpose(Binance widget). The widget doesn't have solana and we don't need the local pngs for our wallet, component updater brings them to us. https://github.com/brave/brave-core-crx-packager/pull/271/files

Reply:
Oh I see, so https://github.com/brave/brave-browser/issues/23934 actually needs to be fixed in component updater rather than core.

Reply:
i have resolved it, LMK if something needed to be done now, (preferably quick solution)